src/le.py

```diff
@@ -2035,6 +2069,8 @@ def __init__(self):
         # Enabled fine-grained logging
         self.debug = False
         # All recognized events are logged
+        self.debug_cmds = False
```

Comment:
actually that comment is for a different debug option - I inserted a line in the wrong place; will correct this.